README.md

```diff
@@ -73,82 +56,81 @@ If you use multiples registries in your organization and need to fetch packages
 
 ### Override public packages
 
-   If you want to use a modified version of some 3rd-party package (for example, you found a bug, but maintainer didn't accept pull request yet), you can publish your version locally under the same name.
+If you want to use a modified version of some 3rd-party package (for example, you found a bug, but maintainer didn't accept pull request yet), you can publish your version locally under the same name. See in detail each of these [use cases](https://github.com/verdaccio/verdaccio/tree/master/docs/use-cases.md).
 
-See in detail each of these [use cases](https://github.com/verdaccio/verdaccio/tree/master/docs/use-cases.md).
+### E2E Testing
 
-<h2 align="center">Get Started</h2>
+Verdaccio has proved to be a lightweight registry that can be booted in couple of seconds, fast enough for any CI. Many open source projects use verdaccio for end to end testing, to mention some examples, **create-react-app**, **mozilla neutrino**, **pnpm**, **storybook**, **alfresco** or **eclipse theia**. You can read more in your dedicated article to E2E in our blog.
```

Comment:
...be booted in a couple of seconds....

Reply:
You can read more in your dedicated article to E2E in our blog ---> You can read more in dedicated article to E2E in our blog